### ISSUE TITLE
How to use topology to identify whether the C atom is aromatic or not?

### ISSUE THREAD

**Issue by Dadiao-shuai** [mdtraj/mdtraj#1845]
In the following script, the _element type_ is obtained, but I wanna displace normal C and aromatic C for LIG. What should I do? 
My idea is to check its bond, but I do not know what to do.

```
    def parsePDB(self):
        top = self.pdb.topology
        residues = [str(residue)[:3] for residue in top.residues]
        residues_cp = residues.copy()

        # number of all groups in the complex except the ligand
        LIG_number = residues.index('LIG')
        self.residues_indices = [i for i in range(top.n_residues) if i != LIG_number]

        # Get the types of atoms in the protein and ligand
        self.rec_indices = top.select('protein')
        self.lig_indices = top.select('resname LIG')
        table, bond = top.to_dataframe()
        self.all_ele = table['element']
        self.lig_ele = table['element'][self.lig_indices]
```

**Comment by mattwthompson** [comment]
I don't think PDB files contain aromaticity information for ligands. (For proteins and some standard residues this is probably different.) The standard practice is to also carry along some richer description of a ligand, typically a SMILES pattern will be enough.

It looks like RDKit doesn't even try (maybe something has changed since 2016): https://github.com/rdkit/rdkit/issues/1031#issuecomment-240308113

The OpenFF Toolkit handles this by requiring this sort of information when a PDB containing a ligand is loaded; the API might look verbose but I promise everything there is there for a purpose

- Example: https://docs.openforcefield.org/projects/toolkit/en/stable/users/molecule_cookbook.html#from-pdb-file
- Methods:
  - https://docs.openforcefield.org/projects/toolkit/en/stable/api/generated/openff.toolkit.topology.Molecule.html#openff.toolkit.topology.Molecule.from_pdb_and_smiles
  - https://docs.openforcefield.org/projects/toolkit/en/stable/api/generated/openff.toolkit.topology.Topology.html#openff.toolkit.topology.Topology.from_pdb (note the `unique_molecules` argument)

**Comment by sukritsingh** [comment]
> I don't think PDB files contain aromaticity information for ligands

This is correct. The PDB file doesn't have any sort of atom-typing beyond elements, and I think having that kind of atom-typing would be a big overhaul for us and potentially out of the scope of what MDTraj is intended for. OpenFF or openmmforcefields is probably the best way to go here!
